go/cmd/dolt/commands/sqlserver/sqlclient.go

```diff
@@ -140,11 +140,11 @@ func (cmd SqlClientCmd) Exec(ctx context.Context, commandStr string, args []stri
 		}
 	}()
 
-	_ = iohelp.WriteLine(cli.CliOut, `# Welcome to the Dolt MySQL client.
+	_ = iohelp.WriteLine(cli.CliOut, `# Welcome to the Dolt SQL client.
 # Statements must be terminated with ';'.
 # "exit" or "quit" (or Ctrl-D) to exit.`)
 	historyFile := filepath.Join(".sqlhistory") // history file written to working dir
-	prompt := "mysql> "
+	prompt := "doltsql> "
```

Comment:
I’m not sure about this change. All in all, this is a standard MySQL client, and leaving it prompt as `MySQL` better communicates that. If someone fires up the client and sees `doltsql`, it’s not immediately clear that you can connect to _every_ MySQL instance out there.

Reply:
i dont feel strongly, ill revert